### DISCUSSION TITLE
Ability to reorder categories in left-hand bar.

### DISCUSSION THREAD

**Discussion by peterleerental** [music-assistant/support#1578] [Feature requests and ideas]
We mainly listen to classical music and the current order (Home, "Artist", Album, Tracks etc.) is not very useful for classical music. With classical music people are mostly interested in the composer or the name of the album (which usually includes the composer's name). 'Tracks' is almost irrelevant in classical music. The performer is less important and it would be a nice feature to be able to re-order the categories in the left-hand bar to suit our listening preferences. I realise this won't be high priority but it's important enough for me to file a feature request!

**Comment by OzGav** [comment]
Thanks. The UI is a work in progress so this will be considered in due course.

**Reply by OzGav** [reply]
This has been implemented in the latest frontend versions

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
